apitools/gen/gen_client_lib.py

```diff
@@ -223,8 +239,10 @@ def WriteSetupPy(self, out):
         printer('import setuptools')
         printer('REQUIREMENTS = [')
         with printer.Indent(indent='    '):
-            # TODO(craigcitro): Have this track apitools' version.
-            printer('"google-apitools>=0.4.8",')
+            if self.apitools_version.startswith('0.4.'):
+                printer('"google-apitools>=0.4.8,<0.5",')
```

Comment:
do we need to worry about this case? we won't be regenerating old clients with the new code, right?